src/app.ts

```diff
@@ -1,8 +1,18 @@
 import * as dotenv from "dotenv";
 dotenv.config();
 
+import { db } from "@siggame/colisee-lib";
```

Comment:
I would recommend adding some error handling and logging middleware and importing it in.

`app.use(err, req, res, next)` - Error handling (This should be placed after everything)

`app.use(req, res, next)`